docs/src/prepareTransaction.md.ejs

```diff
@@ -1,6 +1,6 @@
 ## prepareTransaction
 
-`prepareTransaction(transaction: object, instructions: object): Promise<object>`
+`prepareTransaction(transaction: [Transaction](https://xrpl.org/transaction-formats.html), instructions?: [Instructions](#transaction-instructions)): Promise<object>`
```

Comment:
Sadly, this doesn't work the way you want it to. The `code` syntax blocks further parsing inside it, so you end up with something that looks like this:

> ![2020-04-05_020114_867340003](https://user-images.githubusercontent.com/7515597/78470860-673d2100-76e1-11ea-8f6f-209e8f082645.png)

Should probably change it back here (but if possible, leave it as-is in the table below)